nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,19 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 2.2.0
+***************
+
+* Added `timeout` parameter to `nrf_modem_trace_get()`.
+* Fixed an issue when compiling the :file:`nrf_modem.h` header in C++.
+* The Delta DFU interface (:file:`nrf_modem_delta_dfu.h`) is now thread safe.
+* Fixed possible race conditions in the `nrf_modem_init()` and `nrf_modem_shutdown()` functions.
+* Fixed a bug in `nrf_listen()` that let the queue of incoming connection requests be of size one.
+* The ``NRF_MODEM_GNSS_EVT_BLOCKED`` event is now sent only when the GNSS stack does not get any runtime due to LTE activity, while earlier it could be sent also when the GNSS stack average runtime was too short.
+* Removed the application EGU interrupt. The library uses only the IPC peripheral interrupt now.
```

Comment:
```suggestion
* Fixed possible race conditions in the :c:func:`nrf_modem_init` and :c:func:`nrf_modem_shutdown` functions.
* Fixed a bug in the :c:func:`nrf_listen` function that allows the queue of incoming connection requests to be of size one.
* The :c:data:`NRF_MODEM_GNSS_EVT_BLOCKED` event is now sent only when the GNSS stack does not get any runtime due to LTE activity, whereas earlier it could be sent when the GNSS stack average runtime was too short.
* Removed the EGU interrupt of the application. The library uses only the IPC peripheral interrupt now.
```

Reply:
why remove _also_ in `[...] it could be sent also when the GNSS`? I think it emphasizes that earlier it could have been sent in both cases, now only in one.

not sure about changing _let_ into _allow_ `a bug .. that let the queue .. be of size one` ->
`a bug .. that allows the queue .. to be of size one`

The queue can be of size one under normal operation. It's always allowed to be of size one, in a sense, if one is the number of pending incoming connections. The bug made it be of size one always.

Reply:
Thanks for the explanation, and after going through this, I would stick by your initial statements. I have updated the comments accordingly.

Reply:
Ok, regarding `Removed the EGU interrupt of the application` I have rephrased it as
`Removed the usage of the application software interrupt.`

Reply:
In theory, it could have been a peripheral other than the EGU to generate the software interrupt, so I think that's clearer.